specs-go/config.go

```diff
@@ -117,6 +117,11 @@ type Mount struct {
 	Source string `json:"source,omitempty"`
 	// Options are fstab style mount options.
 	Options []string `json:"options,omitempty"`
+
+	// UID/GID mappings used for changing file owners w/o calling chown, fs should support it.
+	// Every mount point could have its own mapping.
+	UIDMappings []LinuxIDMapping `json:"uid_mappings,omitempty" platform:"linux"`
+	GIDMappings []LinuxIDMapping `json:"gid_mappings,omitempty" platform:"linux"`
```

Comment:
please use the name `uidMappings` and `gidMappings` here:

```suggestion
	UIDMappings []LinuxIDMapping `json:"uidMappings,omitempty" platform:"linux"`
	GIDMappings []LinuxIDMapping `json:"gidMappings,omitempty" platform:"linux"`
```